src/features/tools/employees/index.tsx

```diff
@@ -0,0 +1,40 @@
+import { EmployeeFilter } from 'features/tools/employees/components/EmployeeFilter'
+import { EmployeeTable } from 'features/tools/employees/components/EmployeeTable'
+import { Row } from 'features/tools/employees/utils'
+import { Card, CardsWrapper } from 'hedvig-ui/card'
+import { MainHeadline } from 'hedvig-ui/typography'
+import React, { useState } from 'react'
+import { connect } from 'react-redux'
+import { BackofficeStore } from 'store/storeTypes'
+
+export const EmployeesComponent: React.FC<{ scopes: readonly string[] }> = (
+  scopes,
```

Comment:
```suggestion
  { scopes },
```

And then just go for `scope` in the rest of the file, instead of `scope.scope`